recipe/meta.yaml

```diff
@@ -16,17 +16,18 @@ build:
 
 requirements:
   host:
-    - python >=3.8
+    - python >=3.9
     - pip
   run:
-    - python >=3.8
+    - python >=3.9
     - dask-core >=2022.08.1
     - distributed >=2022.08.1
     - python-kubernetes >=12.0.1
     - kubernetes_asyncio >=12.0.1
     - kopf >=1.35.3
     - pykube-ng >=22.9.0
     - rich >=12.5.1
+    - kr8s ==0.5.1
```

Comment:
Is this correct? That is quite an old version and we don't have that package in conda-forge yet.

Reply:
Thanks for the review @ocefpaf!

It definitely exists on PyPI https://pypi.org/project/kr8s/0.5.1/ but is not on Conda Forge yet (it's a new package).

Reply:
I've opened https://github.com/conda-forge/staged-recipes/pull/23176 to get things in.

Reply:
Google was throwing me at k8s and I did not even noticed :-/